gensim/test/test_ldamodel.py

```diff
@@ -253,6 +255,12 @@ def testShowTopics(self):
                 self.assertTrue(isinstance(k, six.string_types))
                 self.assertTrue(isinstance(v, float))
 
+        #testing print_topic
```

Comment:
Should I also create a unified test_basemodel.py to handle common tests in all the models like testShowTopic and testShowTopics?